### PR TITLE
[Mesh] Add ability to translate

### PR DESCRIPTION
**Pull request by Kuzma30**
This fix https://github.com/FreeCAD/FreeCAD-translations/issues/53

Thank you for creating a pull request to contribute to FreeCAD! To ease integration, we ask you to conform to the following items. Pull requests which don't satisfy all the items below might be rejected. If you are in doubt with any of the items below, don't hesitate to ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10)!

- [ ]  Your pull request is confined strictly to a single module. That is, all the files changed by your pull request are either in `App`, `Base`, `Gui` or one of the `Mod` subfolders. If you need to make changes in several locations, make several pull requests and wait for the first one to be merged before submitting the next ones
- [ ]  In case your pull request does more than just fixing small bugs, make sure you discussed your ideas with other developers on the FreeCAD forum
- [ ]  Your branch is [rebased](https://git-scm.com/docs/git-rebase) on latest master `git pull --rebase upstream master`
- [ ]  All FreeCAD unit tests are confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ]  All commit messages are [well-written](https://chris.beams.io/posts/git-commit/) ex: `Fixes typo in Draft Move command text`
- [ ]  Your pull request is well written and has a good description, and its title starts with the module name, ex: `Draft: Fixed typos`
- [ ]  Commit messages include `issue #<id>` or `fixes #<id>` where `<id>` is the issue ID number from our [Issues database](https://github.com/FreeCAD/FreeCAD/issues) in case a particular commit solves or is related to an existing issue. Ex: `Draft: fix typos - fixes #4805`

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.20 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=56135).

---

### CODE REVIEW

**Comment by Kuzma30** [comment]
If need I can add commit with changing to .ts file too.

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6938">pipeline status</a> for feature branch [PR_6938](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_6938). Pipeline [551421213](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/551421213) was triggered at [af6b337](https://github.com/FreeCAD/FreeCAD/pull/6938/commits/af6b3374c5585cfada530633f0474c32655d188b). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Mesh/Gui/Workbench.cpp [hunk 1]
```diff
@@ -125,9 +125,9 @@ class MeshInfoWatcher : public Gui::TaskView::TaskWatcher, public Gui::Selection
         if (countPoints > 0) {
             numPoints->setText(QString::number(countPoints));
             numFacets->setText(QString::number(countFacets));
-            numMin->setText(QString::fromLatin1("X: %1\tY: %2\tZ: %3")
+            numMin->setText(tr("X: %1\tY: %2\tZ: %3")
                 .arg(bbox.MinX).arg(bbox.MinY).arg(bbox.MinZ));
-            numMax->setText(QString::fromLatin1("X: %1\tY: %2\tZ: %3")
+            numMax->setText(tr("X: %1\tY: %2\tZ: %3")
```
Comment (chennes) [inline src/Mod/Mesh/Gui/Workbench.cpp:130]: Does it really make sense to translate this?

Comment (Kuzma30) [reply src/Mod/Mesh/Gui/Workbench.cpp:130]: Perhaps for European languages ​​it does not make sense. do you know how, for example, the Arabs write this kind of line? right to left or left to right?

**Pull request merged**